engine/src/main/java/org/hibernate/validator/internal/xml/stax/mapping/ContainerElementTypeConfigurationStaxBuilder.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.internal.xml.stax.mapping;
+
+import java.lang.reflect.Type;
+import java.lang.reflect.TypeVariable;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.hibernate.validator.internal.metadata.aggregated.CascadingMetaDataBuilder;
+import org.hibernate.validator.internal.metadata.core.MetaConstraint;
+import org.hibernate.validator.internal.metadata.location.ConstraintLocation;
+import org.hibernate.validator.internal.xml.ContainerElementTypePath;
+import org.hibernate.validator.internal.xml.binding.ContainerElementTypeType;
+
+/**
+ * Builds the cascading and type argument constraints configuration from the {@link ContainerElementTypeType} elements.
+ *
+ * @author Guillaume Smet
+ * @author Marko Bekhta
+ */
+class ContainerElementTypeConfigurationStaxBuilder {
```

Comment:
I was surprised it did not extend `AbstractStaxBuilder`? I suppose it's not needed?

Reply:
in this case the builder is just a container for the list `List<ContainerElementTypeStaxBuilder> containerElementTypeStaxBuilders` so maybe remove the `Stax` part from the name ?

Reply:
Yes, let's remove the `Stax` part.